lib/torrent.js

```diff
@@ -547,9 +553,10 @@ class Torrent extends EventEmitter {
     })
   }
 
-  _verifyPieces () {
-    parallelLimit(this.pieces.map((_, index) => cb => {
+  _verifyPieces (cb) {
+    parallelLimit(this.pieces.map((piece, index) => cb => {
       if (this.destroyed) return cb(new Error('torrent is destroyed'))
+      if (!piece) return cb(null)
```

Comment:
Why was this line not necessary before yet is necessary now?

Reply:
It's not necessary, just a slight optimization to avoid `store.get` returning an error. I'll take it out.